src/common/datePageGenerator.ts

```diff
@@ -93,5 +93,7 @@ export function queryPaginatedByDate<
       }
       return builder;
     },
+    undefined,
```

Comment:
not sure about this syntax, its gonna look bad

Reply:
i might have an idea for something else, we can try monday

Reply:
Decided to do as separate PR